GLTFSDK.Test/Source/AnimationUtilsTests.cpp

```diff
@@ -224,6 +192,69 @@ namespace Microsoft
                     VerifyGetMorphWeights<int16_t>();
                     VerifyGetMorphWeights<uint16_t>();
                 }
+
+                // Utility for verifying GetRotations
+                template<typename T>
```

Comment:
Why not have 2 template parameters instead of this map?

Reply:
It's just copied from existing code, but I'd say it's a bit more robust with one param since it takes responsibility for the mapping out of the hands of the caller.